src/main/java/no/nav/foreldrepenger/oversikt/oppgave/BrukernotifikasjonBeskjedVedMottattSøknadTask.java

```diff
@@ -0,0 +1,46 @@
+package no.nav.foreldrepenger.oversikt.oppgave;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+import no.nav.foreldrepenger.common.domain.Fødselsnummer;
+import no.nav.foreldrepenger.oversikt.domene.Saksnummer;
+import no.nav.foreldrepenger.oversikt.domene.YtelseType;
+import no.nav.vedtak.felles.prosesstask.api.ProsessTask;
+import no.nav.vedtak.felles.prosesstask.api.ProsessTaskData;
+import no.nav.vedtak.felles.prosesstask.api.ProsessTaskHandler;
+
+import static no.nav.vedtak.felles.prosesstask.api.CommonTaskProperties.SAKSNUMMER;
+
+@ApplicationScoped
+@ProsessTask("dittnav.beskjed")
+public class BrukernotifikasjonBeskjedVedMottattSøknadTask implements ProsessTaskHandler {
+
+    public static final String YTELSE_TYPE = "gjelderYtelsetype";
+    public static final String ER_ENDRINGSSØKNAD = "erEndringssøknad";
+
+    public static final String FØDSELSNUMMER = "fødselsnummer";
+    private static final String TEMPLATE_ENDRINGSSOKNAD = "Vi mottok en søknad om endring av %s";
+    private static final String TEMPLATE_FØRSTEGANGSSOKNAD = "Vi mottok en søknad om %s";
+    private BrukernotifikasjonTjeneste brukernotifikasjonTjeneste;
+
+    @Inject
+    BrukernotifikasjonBeskjedVedMottattSøknadTask(BrukernotifikasjonTjeneste brukernotifikasjonTjeneste) {
+        this.brukernotifikasjonTjeneste = brukernotifikasjonTjeneste;
+    }
+
+    @Override
+    public void doTask(ProsessTaskData data) {
+        var fnr = new Fødselsnummer(data.getPropertyValue(FØDSELSNUMMER));
+        var saksnummer = new Saksnummer(data.getPropertyValue(SAKSNUMMER));
+        var tekst = tekst(data);
+        brukernotifikasjonTjeneste.sendBeskjed(tekst, fnr, saksnummer);
+    }
+
+    private static String tekst(ProsessTaskData data) {
+        var ytelseType = YtelseType.valueOf(data.getPropertyValue(YTELSE_TYPE));
+        var erEndringssøknad = Boolean.parseBoolean(data.getPropertyValue(ER_ENDRINGSSØKNAD));
+        var template = erEndringssøknad ? TEMPLATE_ENDRINGSSOKNAD : TEMPLATE_FØRSTEGANGSSOKNAD;
+        return String.format(template, ytelseType.name().toLowerCase());
```

Comment:
Kanskje tryggest å bruke BrukernotifikasjonTjeneste.ytelsetype for å finne ytelsenavn

Reply:
Joda, vurderte den. Det innebærer at jeg flytter teksten inn i tjenesten. Kan gjøre det.